packages/renderers/renderer-svelte/index.js

```diff
@@ -13,7 +14,16 @@ export default {
       plugins: [
         svelte({
           emitCss: true,
-          compilerOptions: { hydratable: true },
+          compilerOptions: { dev: mode === 'development', hydratable: true },
+          preprocess: [
+            preprocess({
```

Comment:
So these are actually configurable if the user adds a `svelte.config.js` file, I believe. Should we add docs somewhere?

Reply:
But I think these are safe defaults because it just helps Svelte match what Astro and other frameworks do out-of-the-box.